### DISCUSSION TITLE
Control Camera speed in interpolation

### DISCUSSION THREAD

**Discussion by dhruvadhia1** [google/model-viewer#2131] [Ideas]
Is it possible to control the camera speed in this eg ?

https://modelviewer.dev/examples/stagingandcameras/#interpolation

for eg slow down the camera interpolation speed.

**Comment by elalish** [comment]
Not yet, but it's been requested before. It may be time to expose it. What we can control is a time-constant (currently 50ms), since interpolation is not a constant speed, but an acceleration and deceleration. Would that be sufficient?

**Reply by dhruvadhia1** [reply]
Yes, that would be an interesting effect. Have a look at this example https://weedensenteret.no/, especially the annotations on the table 3D model. The transitions are nice due to the acceleration effect. So yes acceleration and deceleration will be good to expose.
